app/helpers/elasticsearch_helper.rb

```diff
@@ -5,6 +5,7 @@ module ElasticsearchHelper
   # TODO(tiago): Eventually, this filtering should be moved to ElasticSearch, at which point we should reduce the max value
   # (We can also move it to a parameter that clients of this functions can set)
   MAX_SEARCH_RESULTS = 50
+  HOMO_SAPIEN_TAX_ID = 9606
```

Comment:
Can you create a common place to put this constant? 
I wonder where it makes more sense... `TaxonLineage`?

Reply:
Yes, good idea. And looking now there is actually a `HOMO_SAPIENS_TAX_ID` already in `TaxonLineage`... which contains two IDs!

Any idea why? And why this constant wasn't used in report helper?

Reply:
Not sure... probably missed at the time...